luskan/src/main/webapp/index.html

```diff
@@ -0,0 +1,165 @@
+    jenkinsd 2016.03.01
+    */
+    .swagger-section #header {
+      background-color: rgba(42, 125, 166, 0.7);
+    }
+
+    .swagger-section #header #logo {
+      padding: 0;
+    }
+
+    .swagger-section #header a#logo {
+      background: transparent;
+    }
+  </style>
+
+  <script type="text/javascript">
+    $(function ()
+      {
+        var url = "service.yaml";
+
+        hljs.configure({
+                         highlightSizeThreshold: 5000
+                       });
+
+        // Pre load translate...
+        if (window.SwaggerTranslator)
+        {
+          window.SwaggerTranslator.translate();
+        }
+        window.swaggerUi = new SwaggerUi({
+          url: url,
+          dom_id: "swagger-ui-container",
+          supportedSubmitMethods: ['get', 'post', 'put', 'delete', 'patch'],
+          onComplete: function (swaggerApi, swaggerUi)
+          {
+            if (typeof initOAuth == "function")
+            {
+              initOAuth({
+                          clientId: "your-client-id",
+                          clientSecret: "your-client-secret-if-required",
+                          realm: "your-realms",
+                          appName: "your-app-name",
+                          scopeSeparator: ",",
+                          additionalQueryStringParams: {}
+                        });
+            }
+
+            if (window.SwaggerTranslator)
+            {
+              window.SwaggerTranslator.translate();
+            }
+
+            // addApiKeyAuthorization();
+          },
+          onFailure: function (data)
+          {
+            log("Unable to Load SwaggerUI");
+          },
+          docExpansion: "list",
+          jsonEditor: false,
+          apisSorter: "alpha",
+          defaultModelRendering: 'schema',
+          showRequestHeaders: true,
+          validatorUrl: null
+        });
+
+        function addApiKeyAuthorization()
+        {
+          var key = encodeURIComponent($('#input_apiKey')[0].value);
+          if (key && key.trim() != "")
+          {
+            var apiKeyAuth = new SwaggerClient.ApiKeyAuthorization("api_key", key, "query");
+            window.swaggerUi.api.clientAuthorizations.add("api_key", apiKeyAuth);
+            log("added key " + key);
+          }
+        }
+
+        $('#input_apiKey').change(addApiKeyAuthorization);
+
+        // if you have an apiKey you would like to pre-populate on the page for demonstration purposes...
+        /*
+         var apiKey = "myApiKeyXXXX123456789";
+         $('#input_apiKey').val(apiKey);
+         */
+
+        window.swaggerUi.load();
+
+        function log()
+        {
+          if ('console' in window)
+          {
+            console.log.apply(console, arguments);
+          }
+        }
+      });
+  </script>
+</head>
+
+<body class="swagger-section">
+<div id='header'>
+  <div class="swagger-ui-wrap">
+    <a id="logo" href="${request.servletPath}">Storage Inventory Service (luskan) API Doc</a>
```

Comment:
There is no one "Storage Inventory Service" -- make this

luskan API: Storage Inventory Query Service